ruby_on_rails/configure_ci.md

```diff
@@ -13,6 +13,8 @@ the organisation. That's not a problem, you can configure it anyway and ask wg-o
 the organisation afterwards (you have 100 free builds)
 * Create a project here for the branch `master`: <https://semaphoreci.com/organizations/renuo/projects/new>
 * Skip the automated project analysis and choose `JavaScript` as *Language* with the newest version available.
+* Add the Renuo team to the collaborators, so that everyone can see the build statuses on SemaphoreCI.
+![image](https://user-images.githubusercontent.com/31915276/56659136-a8891c00-669c-11e9-9836-58c478354fbc.png)
```

Comment:
is it common here that the attachments are not stored inside this project?

Reply:
I didn't store them inside the project before 🤔. I will from now on, when the picture has to be displayed inside the readme.